src/frontend/src/icp/components/fee/EthereumEstimatedFee.svelte

```diff
@@ -15,19 +15,28 @@
 	import { isTokenCkErc20Ledger } from '$icp/utils/ic-send.utils';
 	import { ethereumFeeTokenCkEth } from '$icp/derived/ethereum-fee.derived';
 	import { tokenWithFallbackAsIcToken } from '$icp/derived/ic-token.derived';
+	import { balancesStore } from '$lib/stores/balances.store';
+	import type { Token } from '$lib/types/token';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
 
 	let ckEr20 = false;
 	$: ckEr20 = isTokenCkErc20Ledger($tokenWithFallbackAsIcToken);
 
 	const { store } = getContext<EthereumFeeContext>(ETHEREUM_FEE_CONTEXT_KEY);
 
+	let feeToken: Token;
+	$: feeToken = ckEr20 ? $ethereumFeeTokenCkEth ?? $ckEthereumNativeToken : $ckEthereumNativeToken;
+
 	let feeSymbol: string;
-	$: feeSymbol = ckEr20
-		? $ethereumFeeTokenCkEth?.symbol ?? $ckEthereumNativeToken.symbol
-		: $ckEthereumNativeToken.symbol;
+	$: feeSymbol = feeToken.symbol;
 
 	let maxTransactionFee: bigint | undefined | null = undefined;
 	$: maxTransactionFee = $store?.maxTransactionFee;
+
+	const balance = $balancesStore?.[feeToken?.id]?.data ?? BigNumber.from(0n);
```

Comment:
Isn't `EthereumEstimatedFee` duplicating the same code, minus the fee to compare, as `FeeDisplay`? Are yo going to refactor that in another PR or?

Reply:
yes, I was thinking in another PR. You prefer here?

Reply:
please check last commit @peterpeterparker

Reply:
Done with PR #1930